z:  add `-c` parameter to the command description (#5262)
tldr-pages/tldr: pages/common/z.md

# z

> > Tracks the most used (_frecent_, read frequently recent) directories and enables quickly navigating to them using string or regex patterns.

Reviewer: Though helpful, I'm not sure tldr-pages in the place to store this info.  The more information link is enough IMO&mdash;but I can be overruled.  Thoughts @navarroaxel?
```suggestion
> Tracks the most used directories and enables quickly navigating to them using string or regex patterns.
```

Reviewer 2: What about this?
```suggestion
> Tracks the most used (by frecency) directories and enables quickly navigating to them using string or regex patterns.
```

- Remove the current directory from `z`'s database:

`z -x .`

> - Go to the directory that is a subdirectory of the current directory.

Reviewer: @crs1138 it appears that `z -c` is the equivalent of `cd`.  If **and only if** this is true, please apply this suggestion:
```suggestion
- Go to the directory that is a subdirectory of the current directory (equivalent of `cd`):
```

Author: Thanks for your feedback. I've applied the changes you requested. Not sure how to proceed now. Can I resubmit this PR or better to make a new one?
